CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/infomap/InfomapBase.java

```diff
@@ -336,9 +336,9 @@ private void tryIndexingIteratively() {
         while (tryIndexing) {
             if (verbose) {
                 if (config.verbosity > 0) {
-                    System.out.printf("Trying to find super modules... ");
+                    System.out.print("Trying to find super modules... ");
```

Comment:
I'll assume this is going to be converted to a LOGGER call in a different PR.

Reply:
There is a separate rule for converted sysout prints to logs and I've opted to save that for a later point